source/reference/procedures.txt

```diff
@@ -0,0 +1,69 @@
+.. _procedures:
+
+==========
+Procedures
+==========
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Procedures visually separate multi-step processes into individual steps.
+Use procedures whenever stepped instructions require any of the following:
+
+- code blocks
+- significant explanation (more than a sentence)
+- images
+- substeps
+- lists
+
+Procedure Types
+---------------
+
+The following table shows the different styles of procedure and the
+syntax required to generate those styles. If you omit the ``style``,
+procedures default to a value of ``connected``.
+
+.. list-table::
+   :widths: 60 40
+   :header-rows: 1
+
+   * - Markup
+     - Result
+
+   * - .. literalinclude:: /includes/procedure-example-default.rst
+          :copyable: false
+     - .. include:: /includes/procedure-example-default.rst
+     
+
+   * - .. literalinclude:: /includes/procedure-example-connected.rst
+          :copyable: false
+     - .. include:: /includes/procedure-example-connected.rst
+
+   * - .. literalinclude:: /includes/procedure-example-normal.rst
+          :copyable: false
+     - .. include:: /includes/procedure-example-normal.rst
+
+   * - .. literalinclude:: /includes/procedure-example-compact.rst
+          :copyable: false
+     - **compact style does not render on non-landing pages**
```

Comment:
Suggestion: Can an example from a docs repo be linked here? This left me wondering what it actually looks like when on a landing page.

Reply:
Looks like this isn't in use anywhere yet -- but I added a link to a Realm SDK docs example procedure use in practice.